fdbsyncd/fdbsync.cpp

```diff
@@ -43,6 +43,66 @@ FdbSync::~FdbSync()
     }
 }
 
+
+// Check if interface entries are restored in kernel
+bool FdbSync::isIntfRestoreDone()
+{
+    vector<string> required_modules = {
+            "vxlanmgrd",
+            "intfmgrd",
+            "vlanmgrd",
+            "vrfmgrd"
+        };
+
+    for(string& module : required_modules)
+    {
+        WarmStart::WarmStartState state;
+        
+        WarmStart::getWarmStartState(module, state);
+        if(state == WarmStart::REPLAYED || state == WarmStart::RECONCILED)
+        {
+            SWSS_LOG_INFO("Module %s Replayed or Reconciled %d",module.c_str(), (int) state);            
+        }
+        else
+        {
+            SWSS_LOG_INFO("Module %s NOT Replayed or Reconciled %d",module.c_str(), (int) state);            
+            //return false;
+            // Return true till all the dependant code is checked in
+            return true;
+        }
+    }
+    
+    return true;
+}
+
+// Check if vxlan entries are re-conciled to kernel 
+bool FdbSync::isReadyToReconcile()
+{
+    vector<string> required_modules = {
+        "orchagent",
+    };
+
+    for(string& module : required_modules)
+    {
+        WarmStart::WarmStartState state;
+        
+        WarmStart::getWarmStartState(module, state);
+        if(state == WarmStart::RECONCILED)
+        {
+            SWSS_LOG_INFO("Module %s Reconciled %d",module.c_str(), (int) state);            
+        }
+        else
+        {
+            SWSS_LOG_INFO("Module %s NOT Reconciled %d",module.c_str(), (int) state);            
+            //return false;

```

Comment:
Remove unused code.

Reply:
This is stubbed code so that the basic functionality will not fail until all the warm-reboot changes are available in the code base. Once all the warm-reboot changes are available, the actual code will be uncommented and the stub will be deleted

Reply:
Same points here, the code should be correct, and dependencies should be marked as PR description level and let the test to guide if PR is ready or not.

Reply:
Will update the actual code flow as this is the last dependent PR.